blog/_posts/2023-01-10-new-ab-community-rep.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog-detail
+post-type: blog
+by: Chris Kipp, Scala Center Advisory Board Chair
+title: Eugene Yokota Joins the Scala Center’s Advisory Board as a new Community Representative
+description: Announcement of Eugene Yokota joining the Scala Center Advisory Board as the new Community Representative
+---
+
+On behalf of the Scala Center Advisory Board I’d like to welcome Eugene Yokota
+as a new community representative to the advisory board.
+
+During the past year following [The Community Delegate Terms
+proposal][delegate-terms] both of our previous community reps, [Bill
+Venners][bill-announcement] and [Rob Norris][rob-announcement], stepped down in
+order to make way for the next generation of community reps. I’d like to extend
+the board's gratitude for the tremendous dedication and support both Bill and
+Rob offered the board over the years. Bill has served the board since 2016 and
+Rob since 2019.
+
+In light of the Community Delegate Terms proposal, four out of eleven Scala
+Center Advisory board members formed a working group to review recommendations
+from the board for the next community rep. 
+
+Some key questions that were considered when looking at the recommendations were:
+
+Does this person:
+
+- have a history of contributions to the Scala community? (technical and/or non-technical)
+- have a history of community engagement?
+- bring a diverse perspective and representation to the board?
+- have a collaborative mindset?
+- believe in the values and mission of the Scala Center?
+- have a broad interest in the Scala community as a whole?
+
+We considered these questions in light of what the expectation for a community
+representative are:
+
+- Spokesperson for the Scala community at Advisory Board quarterly meetings
+- Gathering and expressing community’s opinion on certain topics related to Scala Center proposals
+- Point of contact for community members/groups to review and help shaping community proposals
+- Presents and defends community proposals at AB meetings
+
+With all this in mind last November we made the decision on the nomination, and
+we’re happy to introduce Eugene, who has been a pillar in the Scala community
```

Comment:
Thanks for this introduction. A pillar sounds too much, maybe "an unusual enthusiast"?

Reply:
I have no problem with the word pillar, but you're 100% free to veto me 😆 ... seeing that this is you we're talking about. How about `a dedicated enthusiast`? Would that be ok with you?

Reply:
Yea. I think that's better. Thanks!